### ISSUE TITLE
Cloud backup

### ISSUE THREAD

**Issue by geekpete** [lordi/tickmate#67]
Backup to cloud.
B2 would be a nice integration but simple is best first like to email or s3

**Comment by lordi** [comment]
Currently tickmate has no access to internet for privacy reasons, but this will probably change in the future when we have *runtime permissions*.

**Comment by lordi** [comment]
I don't think we should close this issue. Cloud backup might be an important feature which can play well with privacy concerns by using runtime permissions.

**Comment by PanderMusubi** [comment]
How about a daily backup to a file that doesn't have a date in the name. That file can be synced to cloud storage with e.g. syncthing.
